CONTRIBUTING.md

```diff
@@ -1,25 +1,4 @@
-# Contributing
+# Contributing to imitation
```

Comment:
If we're going to make `CONTRIBUTING.md` just a link-post, then I think may as well delete it and reference the docs guide directly from the `README.md`.